src/libs/Accessibility/moveAccessibilityFocus/types.ts

```diff
@@ -0,0 +1,6 @@
+import {ElementRef, RefObject} from 'react';
+import {HostComponent} from 'react-native';
+
+type MoveAccessibilityFocus = (ref?: ElementRef<HostComponent<unknown>> & RefObject<HTMLOrSVGElement>) => void;
```

Comment:
@kubabutkiewicz Did you check with Robert Kozik about this one? You can also post in CK/SWM dev channel.

Reply:
Yes, he said that's ok for him 😄